package.json

```diff
@@ -14,7 +14,7 @@
   "bugs": {
     "url": "https://github.com/true-myth/true-myth/issues"
   },
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
Let's revert this change; I'll make the bump when we actually publish it. Thank you for recognizing the need, though!